<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1331</width>
    <height>848</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimpleChat</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 85, 85);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>321</width>
      <height>841</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(49, 49, 49);
</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_menu_accaunt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PT Sans</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(17, 17, 17);</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>        Чаты</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>        Чаты</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>       Выйти</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(29, 29, 29);</string>
    </property>
    <property name="text">
     <string>           ГРУППЫ</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>281</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(70, 70, 70);
background-color: rgb(58, 58, 58);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(35, 35, 35);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>      ЛИЧНЫЕ ЧАТЫ</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>281</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(70, 70, 70);
background-color: rgb(58, 58, 58);</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_info_group">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>1001</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>23</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(53, 53, 53);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Название чата#</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_add_group">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>490</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(18, 18, 18);</string>
    </property>
    <property name="text">
     <string>Создать группу</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_search_chat">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="proxyButton_search_chat">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send_message">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>790</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Отправить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_message">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>790</y>
      <width>871</width>
      <height>51</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>85</green>
          <blue>85</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>85</green>
          <blue>85</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>85</green>
          <blue>85</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>85</green>
          <blue>85</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>85</green>
          <blue>85</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>85</green>
          <blue>85</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>85</green>
          <blue>85</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>112</red>
          <green>112</green>
          <blue>112</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>85</green>
          <blue>85</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>85</green>
          <blue>85</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="placeholderText">
     <string>Сообщение...</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>991</width>
      <height>691</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
border-radius: 7px;
background-color: #2C313C;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="tabKeyNavigation">
     <bool>false</bool>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>true</bool>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::NoDragDrop</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerItem</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerItem</enum>
    </property>
    <property name="movement">
     <enum>QListView::Static</enum>
    </property>
    <property name="flow">
     <enum>QListView::TopToBottom</enum>
    </property>
    <property name="isWrapping" stdset="0">
     <bool>false</bool>
    </property>
    <property name="resizeMode">
     <enum>QListView::Fixed</enum>
    </property>
    <property name="layoutMode">
     <enum>QListView::SinglePass</enum>
    </property>
    <property name="viewMode">
     <enum>QListView::ListMode</enum>
    </property>
    <property name="uniformItemSizes">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="selectionRectVisible">
     <bool>false</bool>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
